base/essentials.jl

```diff
@@ -148,6 +148,24 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+In the example below the bound check of array A is skipped to improve performance.
+```julia
+    function sum(A::AbstractArray)
+    r = zero(eltype(A))
+    for i = 1:length(A)
+    @inbounds r += A[i]
+    end
+    return r
+    end
```

Comment:
It would just be some minor changes to the `Code`-related method [here](https://github.com/JuliaLang/julia/blob/master/base/markdown/render/terminal/render.jl), so probably simpler to just open a PR with the changes you'd like to propose and then see whether enough people prefer it over what's currently done.